Common/yaml/Contacts.yaml

```diff
@@ -0,0 +1,374 @@
+                
+       '500':
+         description: Internal Server Error
+         content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "Server Error"
+                  
+    delete:
+      tags:
+          - contacts
+      summary: Delete contacts by id
+      description: Delete details of a contact
+      operationId: jshd
+      parameters:
+        - name: id
+          in: path
+          description: Update specific contact's details
+          required: true
+          schema:
+            type: integer
+            default: 10001
+            enum: 
+             - 10001
+            
+          
+      responses:
+       '200':
+          description: successful
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "Contact Delete successfully"
+
+       '404':
+          description: Contact not found
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: Contact not found
+          
+       '500':
+          description: Internal server Error
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: "Internal Server Error"
+                        
+                  
+    get:
+      tags:
+        - contacts
+      summary: Get detils by contactId
+      description: Get details of specific contact
+      operationId: detailsOfcontact
+      parameters:
+        - name: id
+          in: path
+          description: page value for contact
+          required: true
+          schema:
+            type: integer
+            example: 1001
+            
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Contact'
+                  
+        '400':
+          description: BAD REQUEST
+          content:
+            application/json:
+              schema:
+                type: string
+                example:
+                  message: Invalid ContactId
+        '404':
+             description: NOT FOUND
+             content:
+               application/json:
+                 schema:
+                   type: string
+                   example:
+                      message: Contact Not Found with ContactId     
+        '500':
+            description: Internal Server Error
+            content:
+              application/json:
+                 schema:
+                   type: string
+                   example:
+                     message: Server Error        
+components:
+  schemas:
+    contactList:
```

Comment:
Make the schema names consistent. All should either start with capital letter or small letter.

Reply:
fixed please review